test/development/acceptance/ReactRefreshLogBox.test.ts

```diff
@@ -821,17 +822,16 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox %s', () => {
     )
     const { session, browser } = sandbox
     await session.assertHasRedbox()
+    const stack = await getRedboxCallStack(browser)
+    // FIXME: the turbopack side snapshot shouldn't include the node:internal frames
+    if (process.env.TURBOPACK) {
+      expect(stack).toInclude('node:')
+    } else {
+      expect(stack).toMatchSnapshot()
+    }
```

Comment:
Can we make this an inline snapshot for both bundlers?

Reply:
Yea I wanted to do it but we're still having the updating snapshot issue with jest. Let me move this out side of this test suite